src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/ContentHelper.CoreClr.cs

```diff
@@ -14,13 +14,26 @@ internal static partial class ContentHelper
     {
         internal static Encoding GetEncoding(HttpResponseMessage response)
         {
-            string charSet = response.Content.Headers.ContentType.CharSet;
+                
+            //ContentType may not exist in response header.
+            string charSet = null;
+            if ( response.Content.Headers.ContentType != null)
+            { 
+                charSet = response.Content.Headers.ContentType.CharSet;
+            }
```

Comment:
Ahh - that's nice and concise!  Thanks! Did you want me to update the PR with changes to that effect, or is just FYI (which is definitely appreciated since I'm a PowerShell guy who knows just enough csharp to hang himself ;) )